src/store/modules/auth.js

```diff
@@ -50,7 +49,7 @@ export default {
 					"/user/signup",
 					{
 						email: data.email,
-						password: data.password
+						password: data.password,
 					},
 					function handleResponse(resData, jwres) {
 						console.log(resData);
```

Comment:
I'm okay w/ these data logs mainly to placate the linter b/c it gives us an excuse to write the handler params consistently in advance. Open to discussion here